src/core/Form/Combobox/ChipList/ChipList.baseStyles.tsx

```diff
@@ -0,0 +1,17 @@
+import { css } from 'styled-components';
+import { defaultThemeTokens as theme } from '../../../theme';
+import { font } from '../../../theme/reset';
+
+export const baseStyles = css`
+  &.fi-chip-list {
+    ${font(theme)('bodyText')}
+    padding-top: 15px;
+  }
+
+  & .fi-chip-list_wrapper {
+    display: inline-flex;
+    flex-wrap: wrap;
+    width: 100%;
+    gap: 10px;
```

Comment:
This might not work as expected if there is no polyfill in place: https://caniuse.com/flexbox-gap

Reply:
This is now using margin + padding combination. As discussed it can in some cases wrap to next line because of that vs to the gap that works smart.
But at this time I think is a valid compromise to not be stopped by small style issue. And if having good fix for this later, it can the be added in separate PR.